### PR TITLE
Updates from Walkthrough (up until CF-alpha)

### PR DESCRIPTION
**Pull request by jrbudnack**
Some builds to the AWS doc after walking through it, right up to the deployment of the alpha CF.

### CODE REVIEW

File: aws.md [hunk 1]
```diff
@@ -1663,7 +1672,7 @@ ensuring that you get no errors (no output is a good sign).
 
 Then, you can deploy to your BOSH Director via `make deploy`.
 
-Once you've deployed, you can validate the deployment via `bosh deployments`. You should see the bolo deployment. You can find the IP of bolo vm by running `bosh vms` for bolo deployment. In order to visit the Genesis web interface on your `bolo/0` VM from your browser on your laptop, you need to setup port forwarding to enable it.
+Once you've deployed, you can validate the deployment via `bosh deployments`. You should see the bolo deployment. You can find the IP of bolo vm by running `bosh vms` for bolo deployment. In order to visit the Gnossis web interface on your `bolo/0` VM from your browser on your laptop, you need to setup port forwarding to enable it.
```
Comment (7hunderbird) [inline aws.md]: Thanks for reverting this back.  I make some changes before and didn't realize `Gnossis` was a thing.

File: aws.md [hunk 2]
```diff
@@ -2346,7 +2363,37 @@ And once complete, run the smoke tests for good measure:
 
 ```
 $ genesis bosh run errand smoke_tests
-FIXME output
+jbudnack@ip-10-10-0-200:~/ops/cf-deployments/bosh-lite/alpha$ genesis bosh run errand smoke_tests
```
Comment (7hunderbird) [inline aws.md]: Can you change the output of `jbudnack@ip-10-10-0-200:` to a `$` prompt.  Our goal in some of the recent passes is to have the prompts be consistent.

Comment (jrbudnack) [inline aws.md]: Done!

File: aws.md [hunk 3]
```diff
@@ -2363,7 +2363,6 @@ And once complete, run the smoke tests for good measure:
 
 ```
 $ genesis bosh run errand smoke_tests
-jbudnack@ip-10-10-0-200:~/ops/cf-deployments/bosh-lite/alpha$ genesis bosh run errand smoke_tests
```
Comment (7hunderbird) [inline aws.md]: We still like to see the command.  
```
jbudnack@ip-10-10-0-200:~/ops/cf-deployments/bosh-lite/alpha$ genesis bosh run errand smoke_tests
```
becomes
```
$ genesis bosh run errand smoke_tests
```

**Pull request merged**